gcsa/event.py

```diff
@@ -44,7 +46,7 @@ class Transparency:
 class Event(Resource):
     def __init__(
             self,
-            summary: str,
+            summary: str | None,
```

Comment:
I'm avoiding using the new type annotation style for Python < 3.10 compatibility. (Eventually, we'll convert it when p3.9 reaches its end-of-life).

I have also been avoiding using `Optional` (which is open to criticism :)) for brevity reasons as all "optional" arguments have `None` as default. Except `summary`. So we should make it explicit. Let's keep it consistent with everything else and use `summary: str = None`.

Reply:
So my main argument against `Optional` (for other fields) is that it looks better without it in [docs](https://google-calendar-simple-api.readthedocs.io/en/latest/code/event.html) and all optional arguments have `None` as default so (I think and hope that) it's obvious.

But now that I think about it, for the `summary` I'd keep `Optional` without the default parameter to somehow indicate a weird behavior of the API. So let's keep it as you have it now.